src/common/low_precision_transformations/src/fake_quantize_decomposition.cpp

```diff
@@ -396,6 +401,8 @@ bool FakeQuantizeDecompositionTransformation::transform(TransformationContext& c
         }
     }
 
+    // clear the node that was produced by fuseConvert
+    clear_new_nodes();
```

Comment:
Can you explain why do we need this code? Thanks

Reply:
fuseConvert used in FakeQuantizeDecompositionTransformation fuses FQ->Convert to TypeRelaxed FQ. So if FakeQuantizeDecompositionTransformation returns before call fq_decomposition::decomposeFakeQuantize, that TypeRelaxed FQ was not registered that prevented other fusions in the graph rewrite. Here we clear the node produced by fuseConvert since fq_decomposition::decomposeFakeQuantize registers new node itself.

Reply:
Ok, I got it. Thanks for the detailed explanation